ui/packages/atlasmap/src/UI/dialogs/ParametersDialog.stories.tsx

```diff
@@ -20,7 +20,7 @@ import React from 'react';
 import { action } from '@storybook/addon-actions';
 
 const obj = {
-  title: 'UI|Dialogs',
+  title: 'Story UI|Dialogs',
```

Comment:
Is there a reason to have a separate `Story UI` category in storybook?

Reply:
Can you get this back to `UI|Dialogs`? It created a new upper category `Story UI` in storybook separated from other dialogs, I don't think that's necessary.

Reply:
done